sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_generated/_operations_mixin.py

```diff
@@ -994,7 +985,7 @@ def get_deleted_certificates(
         mixin_instance._config = self._config
         mixin_instance._serialize = Serializer(self._models_dict(api_version))
         mixin_instance._deserialize = Deserializer(self._models_dict(api_version))
-        return mixin_instance.get_deleted_certificates(vault_base_url, maxresults, include_pending, **kwargs)
+        return mixin_instance.get_deleted_certificates(vault_base_url, maxresults, **kwargs)
```

Comment:
I labeled this an autorest issue because autorest generates this method to unify the versioned implementations but apparently fails to consider their different signatures. I'm reluctant to leave the solution as "remember 2016-10-01 must be the default version when you generate code for this library". Is there a way to solve the problem more generally in autorest? The root cause here is arguably that the arguments are all positional. If the optional `maxresults` and `include_pending` were passed as keyword arguments, there would be no `TypeError`. The call might still fail because something downstream (the pipeline?) gets an unexpected argument, but at least that would only occur when the user passed that (optional) argument.

Reply:
I'm trying to come up with a design for this on the autorest side, but in the mean time, (at least for certs) to generate 2016-10-01. Sorry :'(